apps/mosaico/src/Mosaico/Footer.purs

```diff
@@ -34,6 +34,11 @@ footer onStaticPageClick =
         ]
     }
   where
+    externalLink caption url =
+      DOM.a
```

Comment:
Do we want to open it in the same browser tab/window as mosaico?

Reply:
No, it's set to target blank now